.circleci/config.yml

```diff
@@ -760,6 +760,56 @@ jobs:
             echo "== Javascript =="
             ls ./build/js/*.js
 
+  cookiecutter-bootstrap:
+    docker:
+      - image: cimg/python:3.10-node
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/project
+    steps:
+      # Activate docker-in-docker
+      - *checkout_fun
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: Install cookiecutter
+          command: pip install cookiecutter
+      - run:
+          name: Create a project from richie cookiecutter template
+          command: cookiecutter ~/fun --directory cookiecutter --no-input organization=fun
+      - run:
+          name: Enter the newly created project and add a new site "ci"
+          command: |
+            cd fun-richie-site-factory
+            cookiecutter template -o sites/ --no-input site=ci domain=ci.test
+      - run:
+          name: Create env files
+          command: |
+            cp fun-richie-site-factory/env.d/aws.dist fun-richie-site-factory/env.d/aws
+            cp fun-richie-site-factory/env.d/development.dist fun-richie-site-factory/env.d/development
+      - run:
+          name: Create media and db volumes for "ci" site
+          command: |
+            mkdir -p fun-richie-site-factory/data/media/ci
+            mkdir -p fun-richie-site-factory/data/db/ci
+      - run:
+          name: Build front
+          command: |
+            cd fun-richie-site-factory/sites/ci/src/frontend
+            yarn install
+            yarn build-sass-production
+            yarn build-ts-production
+      - run:
+          name: Activate "ci" site
+          command: echo "export RICHIE_SITE=ci" >> $BASH_ENV
```

Comment:
you could set it for the whole job like it's done here for example:
https://github.com/openfun/richie/blob/master/.circleci/config.yml#L380

Reply:
Thanks !